app/test_main.py

```diff
@@ -1,3 +1,16 @@
 from app.main import get_human_age
 
-# write your code here
+def test_should_return_zero_if_cat_and_dog_age_equal_14():
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_should_return_extra_human_year_after_24_animal_years():
+    assert get_human_age(24, 24) == [2, 2]
```

Comment:
Consider using `@pytest.mark.parametrize` to combine similar test cases and avoid repetition. This will make your tests cleaner and more maintainable, as suggested in the checklist.